### PR TITLE
INTF: change format, descriptions under descriptions

### PR DESCRIPTION
**Pull request by larshp**
add all descriptions under a descriptions field

see #54

### CODE REVIEW

**Comment by schneidermic0** [comment]
Sorry, that I haven't taken it up, yet. I would like to double check whether we need more information then descriptions also in for any component. If we only need descriptions, I like your proposal.

If that's the case, another alternative might be not grouping these under `descriptions`, but just add "descriptions" in the end of the field like `methodDescriptions` or `attributeDescriptions`. But I think I prefer your proposal.

**Comment by albertmink** [comment]
- [ ] update JSON schema for CLAS and INTF
- [ ] update example for CLAS and INTF
- [ ] align with internal resources SAPonly

File: file-formats/clas/type/zif_aff_clas_v1.intf.abap [hunk 1]
```diff
@@ -74,18 +74,22 @@ INTERFACE zif_aff_clas_v1 PUBLIC.
...
-      attributes           TYPE zif_oo_aff_types_v1=>ty_component_descriptions,
-      "! <p class="shorttext">Event Descriptions</p>
-      "! Event descriptions
-      events               TYPE zif_oo_aff_types_v1=>ty_events,
-      "! <p class="shorttext">Method Descriptions</p>
-      "! Method descriptions
-      methods              TYPE zif_oo_aff_types_v1=>ty_methods,
+      "! <p class="sharttext"> Descriptions</p>
+      "! Descriptions maintained in SE80
+      BEGIN OF descriptions,
```
Comment (huber-nicolas) [inline file-formats/clas/type/zif_aff_clas_v1.intf.abap:79]: I think we can put the declaration of the "descriptions" structure to 
https://github.com/SAP/abap-file-formats/blob/main/file-formats/typesUsedForClasAndIntf/zif_oo_aff_types_v1.intf.abap

and use refer to this structure in the intf and clas type

Comment (albertmink) [reply file-formats/clas/type/zif_aff_clas_v1.intf.abap:79]: @wurzka is preparing this one.

**Review by huber-nicolas — APPROVED** [review]

**Review by albertmink — APPROVED** [review]
Look's good to me.

**Pull request merged**